bash/standalone/transferXML.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+if [ -z "$1" -o -z "$2" ] ; then
+    echo use "$0 <path_to_input_dir> <path_to_config_file>"
+    exit
+fi
+
+# set path to watch
+DIR="$1"
+# properties 'user' and 'api_endpoint'
+CONFIG_FILE="$2"
+
+source $CONFIG_FILE
+
+# Key 'user' should be included in credentials file
+if [ -z "$user" -o -z "$api_endpoint" ] ; then
+    echo "Not all required config keys are present: 'user' and 'api_endpoint'"
+    exit 1
+fi
+
+LAST_ADDED_FILE="last_added_file.txt"
```

Comment:
Als dit vanuit cron draait wordt dit bestand in de homedir geschreven. Misschien is dat niet de meest handige plek.

Reply:
Zowel de error log als "last file" worden nu in dezelfde directory weggeschreven als het script zelf